.github/workflows/actions.yml

```diff
@@ -304,13 +304,182 @@ jobs:
           name: ${{ env.ARTIFACT_FILE }}
-          path: ${{ env.INSTALL_PATH }}/${{ env.ARTIFACT_FILE }}
+          path: ${{ env.INSTALL_PATH }}/${{ env.ARTIFACT_FILE }}.zip
+          retention-days: 7 # quickly remove test artifacts
+
+  Windows:
+    needs: lint
+    runs-on: windows-${{ matrix.os-version }}
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+
+          - job-name: '32-bit'
+            fftw-arch: 'x32'
+            cmake-arch: 'Win32'
+            os-version: '2019'
+            qt-version: '5.15.2'
+            qt-arch: 'win32_msvc2019'
+            fftw-url: 'ftp://ftp.fftw.org/pub/fftw/fftw-3.3.5-dll32.zip'
+            cmake-generator: 'Visual Studio 16 2019'
+            msvc-year: '2019'
+            vcvars-script: 'vcvars32.bat'
+            chocolatey-options: '--forcex86' # '--forcex86' for 32-bit build
+            use-qtwebengine: 'ON' # might need to be turned off for MinGW
+            ableton-link: 'ON' # might need to be turned off for MinGW
+            artifact-suffix: 'win32' # set if needed - will trigger artifact upload
+            create-installer: ${{ startsWith(github.ref, 'refs/tags/') }}
+            installer-suffix: 'win32-installer'
+
+          - job-name: '64-bit'
+            fftw-arch: 'x64'
+            cmake-arch: 'x64'
+            os-version: '2019'
+            qt-version: '5.15.2'
+            qt-arch: 'win64_msvc2019_64'
+            fftw-url: 'ftp://ftp.fftw.org/pub/fftw/fftw-3.3.5-dll64.zip'
+            cmake-generator: 'Visual Studio 16 2019'
+            msvc-year: '2019'
+            vcvars-script: 'vcvars64.bat'
+            chocolatey-options: '' # '--forcex86' for 32-bit build
+            use-qtwebengine: 'ON' # might need to be turned off for MinGW
+            ableton-link: 'ON' # might need to be turned off for MinGW
+            artifact-suffix: 'win64' # set if needed - will trigger artifact upload
+            create-installer: ${{ startsWith(github.ref, 'refs/tags/') }}
+            installer-suffix: 'win64-installer'
+
+          - job-name: '64-bit MinGW'
+            fftw-arch: 'x64'
+            os-version: '2019'
+            qt-version: '5.15.2'
+            qt-arch: 'win64_mingw81'
+            fftw-url: 'ftp://ftp.fftw.org/pub/fftw/fftw-3.3.5-dll64.zip'
+            cmake-generator: 'MinGW Makefiles'
+            chocolatey-options: '' # '--forcex86' for 32-bit build
+            use-qtwebengine: 'OFF' # might need to be turned off for MinGW
+            ableton-link: 'OFF' # might need to be turned off for MinGW
+            artifact-suffix: 'win64-mingw' # set if needed - will trigger artifact upload
+
+    name: Windows ${{ matrix.job-name }}
+    env:
+      BUILD_PATH: ${{ github.workspace }}/build
+      INSTALL_PATH: ${{ github.workspace }}/build/Install
+      LIBS_DOWNLOAD_PATH: ${{ github.workspace }}/../3rd-party
+      ASIO_URL: 'http://www.steinberg.net/sdk_downloads/asiosdk2.3.zip'
+      ASIO_FOLDER_NAME: ASIOSDK2.3
```

Comment:
these two should be moved down since you only use them once each

Reply:
Just to make sure - I used the url and the folder [directly in this script](https://github.com/supercollider/supercollider/pull/5377/files#diff-2702f6f7d10a1bffbc8596d1a9925b1234bc4c7a6ff4c21ca6b34c13aa75780eR433), is that what you suggested?